src/axom/primal/geometry/BezierCurve.hpp

```diff
@@ -49,6 +49,12 @@ std::ostream& operator<<(std::ostream& os, const BezierCurve<T, NDIMS>& bCurve);
  * The order of a Bezier curve with N+1 control points is N.
  * The curve is approximated by the control points,
  * parametrized from t=0 to t=1.
+ * 
+ * Contains an array of weights to represent a rational Bezier curve.
+ * Nonrational Bezier curves are identified by an empty weights array.
+ * Algorithms for Rational Bezier curves derived from 
```

Comment:
Please add a note that the weights for rational Bezier curves must be strictly positive